.github/workflows/react-lint-testing.yml

```diff
@@ -1,4 +1,4 @@
-name: TestLinting
+name: Branch
```

Comment:
rename the file to `branch` as well like in `holder`?

Reply:
updated in holder to be lint and testing:)